src/transport/subscription-worker/subscription-worker.ts

```diff
@@ -1877,13 +1938,25 @@ const registerClientIfRequired = (event: RegisterEvent) => {
     for (const _client of clientsBySubscriptionKey)
       consoleLog(`Setup PubNub client ping event ${interval} seconds`, _client);
 
-    pingTimeouts[event.subscriptionKey] = setTimeout(
+    pingTimeouts[subscriptionKey] = setTimeout(
       () => pingClients(subscriptionKey),
       interval * 500 - 1,
```

Comment:
Should this be configurable ?  Just like we allow heartbeat timing configuration on the top of presence interval value.  Or it doesn't matter!!?

Reply:
@mohitpubnub interval value or what exactly?

Reply:
Yes, should it be configurable or not?

Reply:
Got it..`subscriptionWorkerOfflineClientsCheckInterval` should work for it.